internal/uidriver/glfw/ui.go

```diff
@@ -580,6 +598,10 @@ func (u *UserInterface) createWindow() error {
 		u.reqHeight = height
 	})
 
+	u.window.SetFocusCallback(func(_ *glfw.Window, focused bool) {
+		u.setIsForeground(focused)
+	})
```

Comment:
Do we need to use `SetFocusCallback`? 

I think checking `u.window.GetAttrib(glfw.Focused)` should work. See `update` function.